src/Integration/State/StateManager.cs

```diff
@@ -189,8 +189,7 @@ private void RestoreBoundProject(ServerViewModel serverViewModel)
                 return;
             }
 
-            // Ordinal comparer should be good enough: http://docs.sonarqube.org/display/SONAR/Project+Administration#ProjectAdministration-AddingaProject
-            ProjectInformation boundProject = serverViewModel.Projects.FirstOrDefault(pvm => StringComparer.Ordinal.Equals(pvm.Key, this.BoundProjectKey))?.ProjectInformation;
+            ProjectInformation boundProject = serverViewModel.Projects.FirstOrDefault(pvm => ProjectViewModel.KeyComparer.Equals(pvm.Key, this.BoundProjectKey))?.ProjectInformation;
```

Comment:
Why is the `KeyComparer` on `ProjectViewModel`? Shouldn't it really be on `ProjectInformation`?